.github/workflows/codeql.yml

```diff
@@ -1,61 +0,0 @@
-name: "CodeQL"
```

Comment:
Didn't Github replace this somehow ?

Reply:
I got rid of it, it was causing pointless failures and we didn't use it anyway, AFAIK. There are better tools now if we want something like that again IIRC.